server/public/styles.css

```diff
@@ -27,3 +27,33 @@ p {
   width: 100%;
   text-align: center;
 }
+
+.form {
+    justify-content: center;
+    width: 100%;
+    align-items: center;
+    display: flex;
+    flex-direction: column;
+}
+
+.submissionRow {
+  width: 40%;
+}
+
+.checkoutCart {
+width: 100%;
+    text-align: center;
+    margin-right: 395px;
+}
+
+.checkoutPrice {
+    width: 100%;
+    text-align: center;
+    margin-right: 339px;
+}
+
+.orderButton {
+  width: 40%;
+  display: flex;
+  justify-content: start;
+}
```

Comment:
See how much of this CSS you can replace with Bootstrap's helper classes
https://getbootstrap.com/docs/4.4/utilities/borders/